### PR TITLE
added info about the up-for-grabs label

### PR DESCRIPTION
**Pull request by mairaw**
Minor change to our contributing guide. Ready for review.

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C26%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609262339406185-1081%5Cworkflow_report.html).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,11 +15,13 @@ In this topic, we'll cover:
 
 ## Process for contributing
 
-**Step 1:** Open an issue describing the article you wish to write and how it relates to existing content.
+**Step 1:** Open an [issue](https://github.com/dotnet/core-docs/issues) describing the article you wish to write and how it relates to existing content.
 The content inside the **docs** folder is organized into sections that are reflected in the Table of Contents (TOC). Define where the topic will be located in the TOC. Get feedback on your proposal. 
 
 You can skip this step for small changes.
 
+You can also look at our [open issues](https://github.com/dotnet/core-docs/issues) list and volunteer to work on the ones you're interested in. We use the [up-for-grabs](https://github.com/dotnet/core-docs/labels/up-for-grabs) label to tag issues we think the community could easily help us out.
```
Comment (stevehoag) [inline CONTRIBUTING.md]: "issues we think the community could easily help us out." I would reword this to something like "issues that would benefit from community contribution." Also, avoid "we", "us", and "our" - this is open source, so presumably the community is part of "us" ;-)

**Comment by stevehoag** [comment]
What's up with the build warning? I looked at the file in question and I don't see any issues with the links, unless I'm missing something. Is it a false positive?

File: CONTRIBUTING.md (hunk 1 above)
Comment (BillWagner) [inline CONTRIBUTING.md]: In other projects, we've used the phrase "first time contributors", so "to tag issues for first time contributors."

And, a suggestion: Do we want to add a paragraph soliciting contributors to review PRs?

Comment (stevehoag) [inline CONTRIBUTING.md]: Wouldn't that imply that community members that have previously contributed aren't eligible?

Comment (BillWagner) [inline CONTRIBUTING.md]: @stevehoag Good point. We haven't heard that feedback, but that may mean people just assumed it meant only those that hadn't contributed before. We were trying to avoid "beginner" or similar terms. Any suggestions on alternative wording?

Comment (stevehoag) [inline CONTRIBUTING.md]: Maybe just drop "first time"? Or "to tag issues open for contribution"?

And yes, soliciting reviewers would be good too.

Comment (mairaw) [inline CONTRIBUTING.md]: Changed the sentence based on your suggestions. Thanks! 

Now for soliciting reviewers I think we need a more drastic change to this guide. I'd actually reorder the topic to have the following sections (just a suggestion):
- Contributing to content
- Contributing to samples
- Reviewing PRs
- DO's and DON'Ts
- Building the docs
- Contributor License Agreement (I just noticed this content went missing, I'll add it back)

Thoughts?

**Comment by mairaw** [comment]
@stevehoag I've noticed that too in a few PRs. I think it's a build issue. Do you want to file a bug on this?

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C27%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609271816425639-1081%5Cworkflow_report.html).

**Comment by stevehoag** [comment]
@mairaw Opened bug 18963 for the build warning issue.

File: CONTRIBUTING.md (hunk 1 above)
Comment (stevehoag) [inline CONTRIBUTING.md]: Sounds good...

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C27%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609271838463158-1081%5Cworkflow_report.html).

**Comment by stevehoag** [comment]
And of course now that I entered a bug the PR published without the build warning ;-)

**Comment by BillWagner** [comment]
I think this is ready to merge.  Any other comments?

**Comment by mairaw** [comment]
@BillWagner there was a suggestion of inviting the community to review PRs too. Did you see my proposal?

**Comment by BillWagner** [comment]
@mairaw Yes, I just :+1: 'd it.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C17%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611170742073990-1081%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by stevehoag** [comment]
LGTM

**Comment by mairaw** [comment]
This one still has the WIP label but I had to rebase to continue my changes.

**Comment by mairaw** [comment]
@BillWagner @rpetrusha @stevehoag added changes to match what ASP.NET has in their docs and also made the changes we discussed the last time where we would move the sections around.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -47,100 +45,117 @@ Also, remove the "wrench" icon (🔧) from the TOC and the file heading, if appl
...
-can explore the code and experiment with their own changes.
+Sample code may fit more than one of these areas. In those cases, place the sample so it matches
+the topics you are covering in your documents. Ask yourself what readers will learn from reading
+your topic. What will they learn from building and running your sample?
 
-All samples should be complete applications or libraries. Where the sample creates a library,
-it should include unit tests or an application that lets readers run the code.
+Each complete sample that you create should contain a `readme.md` file. This file should
+contain a short description of the sample (one or two paragraphs). Your readme.md
+should tell readers what they will learn by exploring this sample. The readme.md file should also contain
```
Comment (BillWagner) [inline CONTRIBUTING.md]: nit: should readme.md be in code style?

**Review by BillWagner — APPROVED** [review]
This LGTM.  Let's :shipit:

I found one nit, on the style for readme.md.

File: CONTRIBUTING.md (hunk 2 above)
Comment (mairaw) [inline CONTRIBUTING.md]: actually, we standardized on using italic for file names to use the same convention ASP.NET had been using for a long time. I'll change and merge.
